core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingRepository.scala

```diff
@@ -62,4 +62,11 @@ class BillingRepository(dataSource: SlickDataSource) {
       )
     }
 
+  /**
+    * Whenever this is called, the status is implied to be CreationStatus.Ready
+    */
+  def updateLandingZoneId(projectName: RawlsBillingProjectName, landingZoneId: UUID): Future[Int] =
```

Comment:
I think we want this to take the status as an argument. For instance, you will want to set the billing project to ERROR if the landing zone creation fails.

Reply:
OK, I see that you are setting the status to ERROR in another way.

Perhaps just rename the method to something clearer? Like `landingZoneCreationCompleted` or …. I don't know, leave it as `updateLandingZoneId`, and use `updateCreationStatus` to set the status? One of the things I'm thinking about is that the Landing Zone team has a ticket to return the landing zone ID when creation is first kicked off, at which point we can store it then and wouldn't want this side effect of the creation status being set.

Reply:
Yep, Jesus has a PR up now to include the landingZoneId in the creation response. I can fold that in with my open PR, after yours has merged. So definitely it would be good to make setting the landing zone ID only set the ID.

https://github.com/DataBiosphere/terra-workspace-manager/pull/886/files